openquake/engine/calculators/hazard/general.py

```diff
@@ -100,9 +100,15 @@ def execute(self):
         distribution, but it can be overridden in subclasses.
         """
         csm = self.composite_model
+        rlzs_assoc = csm.get_rlzs_assoc()
+        # temporary hack
```

Comment:
This is temporary in the sense that in the future the engine calculators will be replaced by the oq-lite calculators that are free from the instantiation problem without any hack.